pkg/kubelet/kuberuntime/helpers_linux.go

```diff
@@ -22,16 +22,16 @@ package kuberuntime
 const (
 	milliCPUToCPU = 1000
 
-	// 100000 is equivalent to 100ms
+	// 100000 is equivalent to 100μs
 	quotaPeriod    = 100000
 	minQuotaPeriod = 1000
 )
 
 // milliCPUToQuota converts milliCPU to CFS quota and period values
 func milliCPUToQuota(milliCPU int64, period int64) (quota int64) {
 	// CFS quota is measured in two values:
-	//  - cfs_period_us=100ms (the amount of time to measure usage across)
-	//  - cfs_quota=20ms (the amount of cpu time allowed to be used across a period)
+	//  - cfs_period_us=100ms (the amount of time to measure usage across given by period, 100μs despite being named "ms")
+	//  - cfs_quota=20ms (the amount of cpu time allowed to be used across a period, μs as well)
```

Comment:
I think this should change the first part of the comment and not add more text.
and
I think usec instead of µs is fine here for consistency